lib/msal-common/test/uCache/AccessTokenEntity.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { expect } from "chai";
+import { AccessTokenEntity } from "../../src/uCache/entities/AccessTokenEntity";
+import { ATValues } from "./cacheConstants";
+
+describe("AccessTokenEntity.ts Unit Tests", () => {
+
+    it("Verify an AccessTokenEntity entity", () => {
+        const at = new AccessTokenEntity();
+        expect(at instanceof AccessTokenEntity);
+    });
+
+    it("Create an AccessTokenCacheEntity entity", () => {
+        let at = new AccessTokenEntity();
+        Object.assign(at, ATValues);
```

Comment:
I'm not sure I understand this test. Seems like its just testing `Object.assign`?

Reply:
I modified this to be more intuitive in a later PR but the gist is this: This verifies that `AccessTokenEntity` is created as expected; The resulting object has a helper API that generates a key, verifying it confirms creation of the Object instead of just a list of values.